app/database_etl/location_utils/location_util_functions.py

```diff
@@ -94,7 +96,28 @@ def row_in_feature_layer(row: pd.Series, feature_layer: FeatureLayer) -> bool:
     pin = Point({"x": row['pin_longitude'], "y": row['pin_latitude']})
     # construct a geometry filter to check if each point is in a disputed area
     pin_filter = intersects(pin)
-    in_disputed_area = len(feature_layer.query(geometry_filter=pin_filter).features) > 0
+
+    continue_query = True
+    retries = 0
+    MAX_RETRIES = 9
+    # Default to setting in_disputed_area = True to ensure we never show pins in disputed area
+    in_disputed_area = True
+    # Make query to determine whether or not the pin is in the disputed area
+    # If the query times out, retry with exponential backoff
+    while continue_query:
+        try:
+            in_disputed_area = len(feature_layer.query(geometry_filter=pin_filter).features) > 0
+            continue_query = False
+        except Exception as e:
```

Comment:
1) remove as e (unless we are gonna use e) 2) can you add the more specific type of exception instead of just Exception?